### ISSUE TITLE
💡 [FEATURE] - Add a Whole New Section called Algorithms

### ISSUE THREAD

**Issue by ritankarsaha** [jfmartinz/ResourceHub#530]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

In the Algorithms tab, i intend to add and jot down and list all the important algorithms necessary for a successful software developer. 

### Additional Context

Apart from that i also wish to add other necessary details regrading study resources and study youtube channels

**Comment by jfmartinz** [comment]
Can you give me the link?

**Comment by ritankarsaha** [comment]
> Can you give me the link?

yeah sure. I intend on integrating mainly the contents of this link in the project.

https://www.geeksforgeeks.org/learn-data-structures-and-algorithms-dsa-tutorial/
https://techdevguide.withgoogle.com/paths/data-structures-and-algorithms/

and a little of my own input as well.

**Comment by ritankarsaha** [comment]
Did u go through the link? Could u assign me the issue? I could then start working on it ASAP.

**Comment by jfmartinz** [comment]
where do you plan to add it?

**Comment by ritankarsaha** [comment]
i wish to create a new section for it, just like the other sections in the website for app dev, web3 etc etc. like its gonna be a separate tab
